src/Eye4web/ZfcUser/Settings/Factory/Mapper/DoctrineORM/UserSettingMapperFactory.php

```diff
@@ -32,7 +32,7 @@ class UserSettingMapperFactory implements FactoryInterface
      */
     public function createService(ServiceLocatorInterface $serviceLocator)
     {
-        $entityManager = $serviceLocator->get('zfcuser_doctrine_em');
+        $entityManager = $serviceLocator->get('Doctrine\ORM\EntityManager');
```

Comment:
oh, the "zf-commons/zfc-user-doctrine-orm" addition should be in "require" in composer.json